com.unity.formats.alembic/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 ### Changed
 ### Fixed
+- Fixed a bug that caused out of project stream files to have the wrong number of material slots.
+- Fixed a bug that caused crashed when the file was of variable topology and there were empty vertex samples. 
```

Comment:
Suggestion:
`- Fixed a bug that was leading the Editor to crash when importing a file with variable topology and empty vertex samples.`